lib/dts-lib.robot

```diff
@@ -2,18 +2,54 @@
 Resource    terminal.robot
 
 
+*** Variables ***
+# Default DTS link for iPXE boot, can be overwritten by CMD:
+${DTS_IPXE_LINK}=               http://boot.dasharo.com/dts/dts.ipxe
+${BOOT_DTS_FROM_IPXE_SHELL}=    ${FALSE}
+
+
 *** Keywords ***
 Boot Dasharo Tools Suite Via IPXE Shell
+    [Documentation]    Boots DTS via iPXE shell by chaining script. Arguments:
+    ...    dts_chain_link: link to the script to chain. This is useful in case
+    ...    the version of the DTS being booted has not been released yet or if
+    ...    a test version is being used. If no link is given - the standard one
+    ...    is being used, that is: http://boot.dasharo.com/dts/dts.ipxe
+    [Arguments]    ${dts_chain_link}
+    # 1) Check and enable network boot, it is disabled by default:
+    Make Sure That Network Boot Is Enabled
+
+    # 2) Enter iPXE shell:
     Enter IPXE
+
+    # 3) Set up net card:
     Write Into Terminal    dhcp net0
     ${out}=    Read From Terminal Until Prompt
     Should Contain    ${out}    ok
     Set DUT Response Timeout    60s
-    Write Bare Into Terminal    chain http://boot.dasharo.com/dts/dts.ipxe\n    0.1
-    Read From Terminal Until    http://boot.dasharo.com/dts/dts.ipxe...
+
+    # 4) Try to boot via the link:
+    Write Bare Into Terminal    chain ${dts_chain_link}\n
```

Comment:
Why did you remove interval argument (`0.1`)? Won't this affect another tests?

Reply:
Good catch, missed this one, fixed [here](https://github.com/Dasharo/open-source-firmware-validation/compare/be03f347bc525cf96d2df3062466fac08489a04f..18a702db97333836ce2084ea95643c7a74e78e78)